streams/iq.m3u

```diff
@@ -5,14 +5,34 @@ http://rtmp1.abnsat.com/hls/arabic.m3u8
 https://5ab29cc78f681.streamlock.net/afarinTV/livestream/playlist.m3u8
 #EXTINF:-1 tvg-id="AlEtejah.iq",Al-Etejah (576p) [Not 24/7]
 http://streaming.aletejahtv.iq:1935/etejah-newLive/smil:etejahNew.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="AlghadeerTV.iq",AlGhadeer (720p) [Not 24/7]
-https://sd343444.vodu.store:3356/live/Alghadeer/index.m3u8
+#EXTINF:-1 tvg-id="AlforatTV.iq",Alforat TV (1080p) 
+http://95.216.180.111:1935/live/10/playlist.m3u8
+#EXTINF:-1 tvg-id="AlGhadTV.eg",Al Ghad (720p) [Not 24/7]
+https://ythls.onrender.com/channel/UCH4UNKBHSNwstan99YDoCVg.m3u8
+#EXTINF:-1 tvg-id="Alhurra.us",Alhurra (720p)
+https://mbn-ingest-worldsafe.akamaized.net/hls/live/2038900/MBN_Alhurra_Worldsafe_HLS/master.m3u8
+#EXTINF:-1 tvg-id="AlhurraIraq.us",Alhurra Iraq (720p)
+https://mbn-ingest-worldsafe.akamaized.net/hls/live/2038899/MBN_Iraq_Worldsafe_HLS/master.m3u8
+#EXTINF:-1 tvg-id="AlImanTV.lb",Aliman TV (240p) [Not 24/7]
+https://svs.itworkscdn.net/alimanlive/imantv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="AlIraqia.iq",Al Iraqia (720p)
 https://cdn.catiacast.video/abr/8d2ffb0aba244e8d9101a9488a7daa05/playlist.m3u8
 #EXTINF:-1 tvg-id="AlIraqiaNews.iq",Al Iraqia News (720p)
 https://cdn.catiacast.video/abr/78054972db7708422595bc96c6e024ac/playlist.m3u8
-#EXTINF:-1 tvg-id="AlIraqiaSport.iq",Al Iraqia Sport (1080p) [Geo-blocked]
-https://au7live.tooliserver.com/live/149075_0.m3u8?session=
+#EXTINF:-1 tvg-id="AlJazeeraChannel.qa",Al Jazeera Arabic (1080p)
+https://live-hls-web-aja.getaj.net/AJA/index.m3u8
+#EXTINF:-1 tvg-id="AlJazeeraDocumentary.qa",Al Jazeera Documentary (1080p) [Geo-blocked]
+https://live-hls-web-ajd.getaj.net/AJD/index.m3u8
+#EXTINF:-1 tvg-id="AlJazeeraMubasher.qa",Al Jazeera Mubasher (1080p)
+https://live-hls-web-ajm.getaj.net/AJM/index.m3u8
+#EXTINF:-1 tvg-id="AlMaarefTV.kw",Al Maaref (350p)
+https://5e74a9d684b2e.streamlock.net/liveTrans/ngrp:channel23_all/playlist.m3u8
+#EXTINF:-1 tvg-id="AlMayadeenTV.lb",Al Mayadeen (576p)
+https://mdnlv.cdn.octivid.com/almdn/smil:mpegts.stream.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="AlNaeemTV.iq",Al-Naeem TV (576p)
+https://nl2.livekadeh.com/hls2/alnaeem_high.m3u8
+#EXTINF:-1 tvg-id="alrabiaa.tv",Al Rabiaa TV (1080p)
+https://cllive.itworkscdn.net/alrabiaatvnewslive/token=nva=1690991331~dirs=1~hash=058a79456eeadb3deb21b/alrabiaatv.smil/playlist.m3u8
```

Comment:
Expires

Reply:
@UltraHDR you didnt check there was a incorrect tvg-id.